app/src/interfaces/select-icon/icons.json

```diff
@@ -819,7 +819,7 @@
 			"edgesensor_low",
 			"flashlight_off",
 			"flashlight_on",
-			"flourescent",
+			"fluorescent",
```

Comment:
scary

Reply:
I believe this is technically correct in terms of Material Icon's naming, as seen here: https://fonts.google.com/icons?selected=Material+Icons+Outlined:flourescent

Reply:
🤦‍♂️. I'll drop and maybe file a bug to them.

Reply:
https://github.com/google/material-design-icons/issues/1208

If I'm reading it correctly, I think they may actually support the corrected spelling but won't drop the incorrect spelling as that would break existing users.

Reply:
Great find! But from what I understood, they didn't actually went through with the correct spelling as the PR wasn't merged. Also tested the current Material Icons Outline font file to unfortunately not work with the correct spelling. 

There is discussion on upgrading to the fairly new Material Symbols (successor to Material Icons) over in #12958, but seems like even that also doesn't have support for the correct spelling: https://fonts.google.com/icons?icon.query=fluorescent

Guess we'll have to revert this change for the time being.

Reply:
Normally, I'd just drop the individual commits and force-push.

But, I see you've added a commit to this PR.

Looks like you do squash merges, so I guess I'll just do reverts and push them and when you squash merge the changes will fall away.

Reply:
Thanks!